### PR TITLE
DynamoDBToS3Operator - change export_time value to utcnow() in system test 

### PR DESCRIPTION
**Pull request by utkarsharma2**
Since datetime.now() returns the local time which can cause issues while running tests from different timezones, replacing it with datetime.utcnow().

cc: @vincbeck @ferruzzi @o-nikolas

### CODE REVIEW

**Review by hussein-awala — APPROVED** [review]
we should wait for approval from @ferruzzi ([details](https://github.com/apache/airflow/pull/31197#issuecomment-1546298870)) before merging

File: tests/system/providers/amazon/aws/example_dynamodb_to_s3.py
Comment (vincbeck) [inline tests/system/providers/amazon/aws/example_dynamodb_to_s3.py]: Please add a buffer to add robustness. I would remove one week to be on the safe side. Dynamodb expects `export_time` to be a date in the past so I would not send the "current" datetime

**Review by vincbeck — CHANGES_REQUESTED** [review]

File: tests/system/providers/amazon/aws/example_dynamodb_to_s3.py
Comment (vincbeck) [inline tests/system/providers/amazon/aws/example_dynamodb_to_s3.py]:
```suggestion
        export_time=datetime.utcnow() - datetime.timedelta(days=7),
```

Comment (utkarsharma2) [reply tests/system/providers/amazon/aws/example_dynamodb_to_s3.py]: Sure, merged your suggestion

**Review by vincbeck — APPROVED** [review]

**Review by ferruzzi — APPROVED** [review]

**Comment by utkarsharma2** [comment]
@vincbeck @ferruzzi Can anyone please confirm if this fix worked? Took a look at the dashboard - https://aws-mwaa.github.io/open-source/system-tests/dashboard.html didn't see it green for `example_dynamodb_to_s3`

**Comment by utkarsharma2** [comment]
@vincbeck Thanks for the response. I see, sure please share when the PR is ready, would love to take a look. Also, please let me know if I can be of any help.

**Pull request merged**
